src/site/filters/liquid.js

```diff
@@ -33,7 +37,7 @@ module.exports = function registerFilters() {
   liquid.filters.timeZone = (dt, tz, format) => {
     if (dt && tz) {
       const timeZoneDate = new Date(dt).toLocaleString('en-US', {
-        timeZone: tz,
+        timeZone: isIE11() ? undefined : tz,
```

Comment:
I have a few questions:

Are you sure this function is the one causing the issue? Have you verified with a stack trace?

As far as I know, these functions are only supposed to be used in our liquid templates.

Simply returning `undefined` seems like it would cause problems on other pages that might use this function.
Have you looked into using a polyfill instead? Some options are mentioned [here](https://stackoverflow.com/questions/54364061/ie-11-throwing-timezone-is-outside-of-valid-range-when-setting-timezone-to).

I don't see any references to this function anywwhere in our liquid templates, but will do some testing to see what happens when I simply remove the function.

Reply:
Hm so this `timeZone` function executes on the server-side during the website compilation, not on the client's browser. I actually don't follow how `window.ActiveXObject` isn't crashing when you try to run it since that would be `undefined` in Node I think.

Reply:
I checked a few event pages in IE11 and they render fine without that function. I don't think it's being used in our liquid templates.

If this function is being used in our login process, I'd suggest moving it elsewhere. How we resolve the bug at that point would no longer be a facilities team concern.

Reply:
@mpelzsherman
- I'm not 100% positive it is but it's the only place that has a reference to setting the `timeZone` property using the `toLocaleString` method.
- Looking at the stack trace in IE 11 yields no useful information as to where this error is coming from.
- It's using `undefined` only on IE 11 (which I have tested in IE 11 and actually get the formatted date back).

@ncksllvn
- That is good information to know, I was unaware the function executed solely on the server-side.  I know the error in the image only displays when trying to set the `timeZone` property using the `toLocaleString` method. It's possible this is not the culprit and maybe I have to investigate more on how this error is occurring.